server/.env.template

```diff
@@ -1,17 +1,10 @@
 NODE_ENV=development
 
-SERVER_PORT=5000
-
-DB_HOST=localhost
-DB_PORT=5432
-DB_NAME=report_bot
-DB_USER=postgres
-DB_PASSWORD=postgres
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/report_bot
 
 SLACK_APP_TOKEN=
 
-# API KEY
 disable_auth=true
-VL_SSO_KEY=sEVfr3pbLWO8IV5NaLCA
-VL_SSO_URL=http://localhost:3456
-required_team_id=000000
+VL_SSO_KEY=Rs82ntvU1NzCMN9VviwG3jTFn
+VL_SSO_URL=https://sso-staging.vacuumlabs.com
+required_team_id=969465 # developers
```

Comment:
I don't know much about VLSSO, but is it okay to put this info in this (public) repo?

Reply:
Well it's just staging SSO, but as good rule of thumb, I would at least not add SSO_KEY.. it can be found in our other private repos

Reply:
I missed this is not private repo. I removed `VL_SSO_KEY`.